src/sentry/incidents/grouptype.py

```diff
@@ -17,6 +18,9 @@
 from sentry.workflow_engine.models.data_source import DataPacket
 from sentry.workflow_engine.types import DetectorGroupKey
 
+COMPARISON_DELTA_CHOICES = [choice.value for choice in ComparisonDeltaChoices]
+COMPARISON_DELTA_CHOICES.append(None)
```

Comment:
this is kind of the sneaky way I was able to have the schema enum for an optionally null field, I'm open to changing it